lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -342,6 +342,11 @@ class SingleExitLoopTransformer {
     return SILUndef::get(type, currentFn->getModule());
   }
 
+  /// Create a tf_tensor_to_i1 instruction with the given value as argument.
+  GraphOperationInst *createTFInt1ToBuiltinInt1(SILBuilder &builder,
```

Comment:
nit: we don't have the notion of a "tf int1" -- it's more of "tensor handle int1". How about a higher level func name like "createTensorToInt1Inst"?